extensions/2.0/Vendor/MSFT_CollisionPrimitives/schema/glTF.MSFT_CollisionPrimitives.collider.schema.json

```diff
@@ -0,0 +1,100 @@
+{
+    "$schema" : "http://json-schema.org/draft-04/schema",
+    "title" : "MSFT_CollisionPrimitives glTF extension",
+    "type" : "object",
+    "description" : "Parameters describing a node's physics collision geometry.",
+    "allOf" : [ { "$ref" : "glTFChildOfRootProperty.schema.json" } ],
+    "properties" : {
+        "collisionSystem": {
+            "type": "array",
+            "description": "An array of strings representing the names of the collision system which this collider belongs to.",
+            "items": {
+                "type": "string"
+            }
+        },
+        "collideWithSystems": {
```

Comment:
What happens in the default case, when neither `collideWithSystems` or `notCollideWithSystems` is specified?

Reply:
It would be, treated equivalently to `collideWithSystems` being an empty set (i.e., it would collide with an object unless that other object has specified a `collideWithSystems` or a `notCollideWithSystems` which prohibits the collision.

This allows you to categorize an object, but without that object having an opinion on what objects it should collide with. e.g., you can have a collider which declares "I'm a collectable powerup", and then other nodes can decide if they should filter it.